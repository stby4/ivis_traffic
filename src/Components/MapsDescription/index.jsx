import React from 'react'

const MapsDescription = () => <div className="description">
    <p>
        Die Visualisierung zeigt mit Hilfe von Karten der Schweiz die Entwicklung der Unfälle im Verlauf der letzten 24 Jahre (bis 2016) aufgesplittet nach Kantonen.
        Dabei kann im Dropdown Menu gewechselt werden zwischen den relativen Werten "Unfälle pro 1000 Fahrzeuge" und den absoluten Werten "Unfälle insgesamt". 
        Zudem lässt die dritte Auswahlmöglichkeit "Zugelassene Fahrzeuge" einen Vergleich der Unfallsentwicklung zur Anzahl Fahrzeugen zu.
        Beim hovern über die Schweizerkarten werden die Daten für den jeweiligen Kanton angezeigt.

    </p>
</div>


export default MapsDescription