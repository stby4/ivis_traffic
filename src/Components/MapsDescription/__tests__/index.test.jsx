import React from 'react'
import renderer from 'react-test-renderer'
import MapsDescription from '../'

test('Snapshot ColorLegend', () => {
    const tree = renderer
        .create(
            <MapsDescription />
        )
        .toJSON()
    expect(tree).toMatchSnapshot()
})