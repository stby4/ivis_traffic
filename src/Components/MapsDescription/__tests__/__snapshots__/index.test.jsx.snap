// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot ColorLegend 1`] = `
<div
  className="description"
>
  <p>
    Die Visualisierung zeigt mit Hilfe von Karten der Schweiz die Entwicklung der Unfälle im Verlauf der letzten 24 Jahre (bis 2016) aufgesplittet nach Kantonen. Dabei kann im Dropdown Menu gewechselt werden zwischen den relativen Werten "Unfälle pro 1000 Fahrzeuge" und den absoluten Werten "Unfälle insgesamt". Zudem lässt die dritte Auswahlmöglichkeit "Zugelassene Fahrzeuge" einen Vergleich der Unfallsentwicklung zur Anzahl Fahrzeugen zu. Beim hovern über die Schweizerkarten werden die Daten für den jeweiligen Kanton angezeigt.
  </p>
  <h4>
    Quellen
  </h4>
  <ul
    className="sources"
  >
    <li>
      <a
        href="https://www.bfs.admin.ch/bfs/de/home/statistiken/kataloge-datenbanken/tabellen.assetdetail.4402685.html"
        rel="noopener noreferrer"
        target="_blank"
      >
        Strassenfahrzeugbestand: Motorfahrzeuge ab 1990
      </a>
      , Bundesamt für Statistik, abgerufen am 29. Mai 2018
    </li>
    <li>
      <a
        href="https://www.bfs.admin.ch/asset/de/px-x-1106010100_101"
        rel="noopener noreferrer"
        target="_blank"
      >
        Strassenverkehrsunfälle: Unfälle mit Personenschaden nach Kanton, Unfallschwere und Unfallort
      </a>
      , Bundesamt für Statistik, abgerufen am 22. Mai 2018
    </li>
  </ul>
</div>
`;
